src/Microsoft.Management.Deployment/AppCatalog.cpp

```diff
@@ -0,0 +1,136 @@
+#include "pch.h"
+#include <AppInstallerRepositorySource.h>
+#include "PackageCatalog.h"
+#include "PackageCatalog.g.cpp"
+#include "PackageCatalogInfo.h"
+#include "FindPackagesResult.h"
+#include "MatchResult.h"
+#include "CatalogPackage.h"
+#include "PackageMatchFilter.h"
+#include "Microsoft/PredefinedInstalledSourceFactory.h"
+#include <wil\cppwinrt_wrl.h>
+
+namespace winrt::Microsoft::Management::Deployment::implementation
+{
+    void PackageCatalog::Initialize(winrt::Microsoft::Management::Deployment::PackageCatalogInfo info, std::shared_ptr<::AppInstaller::Repository::ISource> source)
```

Comment:
Was this file copied and abandoned?  It seems like it should be PackageCatalog.

Reply:
Yeah, git is being really annoying about renames for some reason, not sure what happened. Deleted.